modules/swagger-codegen/src/main/resources/META-INF/services/io.swagger.codegen.CodegenConfig

```diff
@@ -81,3 +81,4 @@ io.swagger.codegen.languages.TypeScriptJqueryClientCodegen
 io.swagger.codegen.languages.TypeScriptNodeClientCodegen
 io.swagger.codegen.languages.UndertowCodegen
 io.swagger.codegen.languages.ZendExpressivePathHandlerServerCodegen
+io.swagger.codegen.languages.LagomScalaApiCodegen
```

Comment:
Better to put it according to alpabetical order

Reply:
Done